pkg/aspectsupport/logger/manager.go

```diff
@@ -68,30 +90,100 @@ func (m *manager) NewAspect(c *aspectsupport.CombinedConfig, a aspect.Adapter, e
 		inputs = c.Aspect.Inputs
```

Comment:
I don't understand what Aspect.Inputs is for. Who fills that in and how does it relate to things like severityAttribute?